packages/app-page-builder-editor/src/components/Editor/Editor.scss

```diff
@@ -54,5 +54,8 @@
     .webiny-pb-page-element-container::after {
       content: none;
     }
+    .webiny-pb-base-page-element-style::after {
```

Comment:
@Pavel910 it should be copied to both `app-page-builder-editor` and `app-page-builder` packages, right?

Reply:
@neatbyte-vnobis You can completely ignore `app-page-builder-editor` package. It is set to `private`, so it's not being published. It's my sandbox for some experiments.